lib/venue.rb

```diff
@@ -1,2 +1,24 @@
 class Venue
+  attr_reader :name,
+              :capacity,
+              :patrons
+
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @patrons = []
+  end
+
+  def add_patron(add)
+     @patrons.push add
+
+  end
+
+  def yell_at_patrons
+    @patrons.each do |patron|
```

Comment:
Nice start here. Remember when we are using `.each` it will always return to us the original array which in this case is `@patrons`. So we would need to set an accumulator that we can shovel into. I would review the `#each lesson`.